students/SLammertink/Lesson03/strformat_lab.py

```diff
@@ -0,0 +1,65 @@
+#! /usr/bin/env python3
+# UW Self paced Lesson 03 string lambda
+
+# Task 1
+''' Write a format string that will take the following four element tuple:
+( 2, 123.4567, 10000, 12345.67)
+and produce:
+'file_002 :   123.46, 1.00e+04, 1.23e+04' '''
+
+StrList = (2, 123.4567, 10000, 12345.67)
+
+def Task1():
+    print(f"file_{StrList[0]:03d} :  {StrList[1]:.2f}, {StrList[2]:.2e}, {StrList[3]:.2e}")
```

Comment:
Good - make sure function and method names are lower case - any with multiple words use underscore to separate.

Reply:
ok thanks will change the names to lower case in the next exercises!